lib/delta-types/delta-types.cabal

```
@@ -1,6 +1,6 @@
 cabal-version:       2.2
 name:                delta-types
-version:             0.1.0.0
+version:             2024.5.5
```

```suggestion
version:             0.2024.5.5
```

Could you add at least one zero in front? Then we can release `1.0.0.0` on Hackage without thinking whether old versioning schemes are upwards-compatible according to the PVP.